integration-tests/README.md

```diff
@@ -68,16 +68,48 @@ can either be 8 or 11.
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
 The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
 
+## Running tests by docker-compose + mvn
+
+Build druid-cluster, druid-hadoop docker images. From root module run maven command:
+```
+mvn clean install -pl integration-tests -P integration-tests -Ddocker.run.skip=true -Dmaven.test.skip=true
+```
+
+Run druid cluster by docker-compose:
+```
+docker-compose -f integration-tests/docker/docker-compose.yml up
+```
+
+Run integration tests:
+```
+mvn verify -pl integration-tests -P integration-tests -Dgroups=batch-index -Djvm.runtime=8 -Pskip-static-checks -Ddruid.console.skip=true -Dmaven.javadoc.skip=true -Ddocker.run.skip=true -Ddocker.build.skip=true
+```
+
+Run s3 | azure | gcs integration tests:
+
+Prepare configuration for integration tests:
+```
+   integration-tests/docker/environment-configs/override-examples/*
+```
+
+Run docker compose for one of group:
+```
+    docker-compose -f integration-tests/docker/docker-compose-s3.yml up
+```
+
+Run maven command to execute tests. 
+> See: Running a Test That Uses Cloud 
+
 ## Tips & tricks for debugging and developing integration tests
 
 ### Useful mvn command flags
 
-- -Dskip.start.docker=true to skip starting docker containers. This can save ~3 minutes by skipping building and bringing 
+- -Ddocker.build.skip=true to skip build druid containers. This can save ~4 minutes to build druid cluster and druid hadoop.
+You need to build druid containers only once, after you can skip docker build step. 
```

Comment:
Can you mention that this is only if there is no code change to Druid. For example, your integration tests that you are writing might actually catch real bugs in the code. Then after you attempt to fix bug in code, you will need to build druid again.

Reply:
added

Reply:
Thanks!